parser/parser.y

```diff
@@ -2982,7 +2982,29 @@ SubSelect:
 	'(' SelectStmt ')'
 	{
 		s := $2.(*stmts.SelectStmt)
-		s.SetText(yylex.(*lexer).src[yyS[yypt - 1].col-1:yyS[yypt].col-1])
+		src := yylex.(*lexer).src
+		// See the implemention of yyParse function
+		lines := yyS[yypt-1].line-1
+		pos := 0
+		if lines > 0 {
+			cnt := 0
+			for ; pos < len(src); pos++{
```

Comment:
http://play.golang.org/p/jH8auHsni_

In my computer, IndexBytes is nearly 2x faster than looping over.